mapit-backend/src/Controller/UserController.php

```diff
@@ -2,20 +2,47 @@
 
 namespace App\Controller;
 
+use App\Repository\UserRepository;
+use App\Entity\User;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 
 class UserController extends AbstractController
 {
     /**
-     * @Route("/user", name="user")
+     * @Route("/user", methods={"GET"})
      */
-    public function index(): Response
-    {
-        return $this->json([
-            'message' => 'Welcome to your new controller!',
-            'path' => 'src/Controller/UserController.php',
+    public function index(): JsonResponse {
+        $users = $this->getDoctrine()->getRepository(User::class)->findAll();
+        $response = [];
+
+        foreach($users as $user) {
```

Comment:
You can outsource this part to a serializer in the next step but to me it looks good so far :)